### PR TITLE
samples: openthread: Add configuration for ipc_radio

### PR DESCRIPTION
**Pull request by canisLupus1313**
This commit ads missing configs for ipc_radio image for thread.

### CODE REVIEW

**Review by ArekBalysNordic — APPROVED** [review]

File: samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+# Debug and assert configuration
+CONFIG_ASSERT=y
+CONFIG_DEBUG_INFO=y
+CONFIG_EXCEPTION_STACK_TRACE=y
+CONFIG_RESET_ON_FATAL_ERROR=n
+CONFIG_REBOOT=n
+
+# IPC
+CONFIG_MBOX=y
```
Comment (piotrkoziar) [inline samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf:42]: I think this `MBOX` config is no longer needed. There will be a warning about it if I remember correctly.

File: samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+CONFIG_IPC_SERVICE=y
```
Comment (piotrkoziar) [inline samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf:43]: This is enabled by default.

**Review by piotrkoziar — APPROVED** [review]

File: samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+CONFIG_BT_BUF_ACL_TX_SIZE=251
+CONFIG_BT_CTLR_DATA_LENGTH_MAX=251
+CONFIG_BT_CTLR_PHY_2M=n
+
+# 802.15.4 configuration
+CONFIG_NRF_802154_SER_RADIO=y
+CONFIG_NRF_802154_ENCRYPTION=y
+CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=2
+
+# Debug and assert configuration
```
[the remark this answers is not in the document]
Comment (canisLupus1313) [reply samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf]: fixed.

File: samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf (hunk 1 above)
Comment (canisLupus1313) [reply samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf]: fixed.

File: samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf (hunk 2 above)
Comment (canisLupus1313) [reply samples/openthread/cli/sysbuild/ipc_radio/boards/nrf54h20dk_nrf54h20_cpurad.conf]: fixed.

**Pull request merged**